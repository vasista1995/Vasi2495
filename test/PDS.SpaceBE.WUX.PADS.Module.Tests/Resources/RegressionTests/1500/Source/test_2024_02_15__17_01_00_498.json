{
  "IdSource": "SPACEACT2:WUX:BE:VC327640C02:ACWUS:D_WireBond:Ball Height:25175:200698694:0",
  "Data_flat_Metadata": {
    "Package": "PG-SOD323-2-1",
    "ParameterClass": "NON_PPAP",
    "Segment": "DS",
    "Module": "M1_ASSY",
    "FourDReport": "*",
    "ChipType": "D0327H00011",
    "SalesName": "BAT 165 E6874",
    "BusinessSegment": "55",
    "WireSize": "Au wire 38um",
    "Operator": "30073142",
    "Baunumber": "70012651",
    "Line": "SOD323",
    "Shift": "C",
    "Process": "WireBond",
    "Lot": "VC327640C02",
    "ParameterName": "Ball Height",
    "ParameterUnit": "um",
    "ParameterOper": "D_WireBond",
    "ParameterFacility": "ACWUS",
    "SiteKey": "WUX",
    "ChannelId": "25175",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
    "CreatedTimestamp": "2023-10-06T15:32:57Z",
    "UpdatedTimestamp": "2023-10-06T15:32:57Z",
    "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
    "ExportedTimestampUtc": "2023-10-06T15:32:57Z",
    "ExportedTimestamp": "2023-10-06T15:32:57Z",
    "SampleTimestampUtc": "2023-10-06T07:32:57Z",
    "SampleTimestamp": "2023-10-06T15:32:57Z",
    "SourceDataLevel": "C",
    "Equipment": "DS-S56",
    "ChannelName": "Ball_Height/SOD323/Au/38um",
    "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
    "RvStoreFlag": "Y",
    "CalcCreated": "10/03/2022 09:22:44",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 4208187,
    "ViolCount": 0,
    "CalculationStrategy": "ND 4.5 Sigma",
    "LdsID": "1500",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 38.0,
        "SpecLow": 19.0,
        "SpecTarget": 28.5,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYYYYYYYYYYYYYYYYYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "30,2",
        "Median": "30,2",
        "Sigma": ",465985898008574",
        "Min": "29,5",
        "Max": "30,8",
        "Q1": "29,875",
        "Q3": "30,55"
      },
      "ControlLimits": {
        "ctrl_mean_low": 24.8896233993906,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 28.9572041233868,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 33.024784847383,
        "CtrlMeanHighEnabled": "Y",
        "ctrl_low": 24.0000978038884,
        "CtrlLowEnabled": "Y",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "Y",
        "ctrl_target": 28.9570619469026,
        "ctrl_high": 33.9140260899169,
        "CtrlHighEnabled": "Y",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_target": 0.583862693698425,
        "ctrl_sigma_high": 2.15608484464821,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 1.70352362448543,
        "ctrl_range_high": 6.21103188423075,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 29.34375,
        "ExtEwmaMV": 29.4146935771202,
        "ExtMsMV": 0.963432989366671,
        "EwmaS": 0.569508599628224,
        "EwmaR": 1.56987744374369,
        "ExtEwmaMVLCL": 27.6002487972675,
        "ExtEwmaMVCenter": 28.9570619469026,
        "ExtEwmaMVUCL": 30.3138750965377,
        "ExtMaMVLCL": 27.9976501679925,
        "ExtMaMVCenter": 28.9570619469026,
        "ExtMaMVUCL": 29.9164737258127,
        "EwmaSLCL": 0.253412611574425,
        "EwmaSCenter": 0.627084202872847,
        "EwmaSUCL": 1.00075579417127,
        "EwmaRLCL": 0.745124738324833,
        "EwmaRCenter": 1.80707928802589,
        "EwmaRUCL": 2.86903383772695,
        "ExtMSMVLCL": 0.00123950931622967,
        "ExtMSMVCenter": 0.58451537944033,
        "ExtMSMVUCL": 2.34802091556625,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 24.0000978038884,
        "RawUal": 33.9140260899169,
        "SigmaLal": -0.208649806129261,
        "SigmaUal": 3.728306995598,
        "RangeLal": -1.0385105018918,
        "RangeUal": 10.7185401439761
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 200698694,
        "Seqnr": 3,
        "Value": 30.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 8,
        "Value": 29.9,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 7,
        "Value": 30.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 6,
        "Value": 30.5,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 5,
        "Value": 29.8,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 4,
        "Value": 30.8,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 2,
        "Value": 30.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200698694,
        "Seqnr": 1,
        "Value": 29.5,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 8
  },
  "ProductionAction": {
    "Id": "SPACEACT1:WUX:BE:ACWUS:D_WireBond:Ball Height:25175",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:VC327640C02",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "WUX:BE:VC327640C02:ACWUS:D_WireBond:Ball Height:25175:200698694:0"
  }
}