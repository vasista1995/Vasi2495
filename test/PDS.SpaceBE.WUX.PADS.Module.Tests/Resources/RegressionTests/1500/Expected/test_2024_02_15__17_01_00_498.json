{
  "Data_flat_Metadata": {
    "Baunumber": "70012651",
    "Line": "SOD323",
    "Shift": "C",
    "Process": "WireBond",
    "MeasLot": "VC327640C02",
    "ParameterOper": "D_WireBond",
    "ParameterFacility": "ACWUS",
    "SiteKey": "WUX",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-06T07:32:57Z",
    "BeginTimestamp": "2023-10-06T15:32:57Z",
    "EndTimestampUtc": "2023-10-06T07:32:57Z",
    "EndTimestamp": "2023-10-06T15:32:57Z",
    "Equipment": "DS-S56"
  },
  "Data_flat_StrangeMetadata": {
    "Package": "PG-SOD323-2-1",
    "SalesName": "BAT 165 E6874",
    "BusinessSegment": "55",
    "Wiresize": "Au wire 38um",
    "ChipType": "D0327H00011",
    "Operator": "30073142"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 3,
          "SampleId": 200698694,
          "Value": 30.0,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 8,
          "SampleId": 200698694,
          "Value": 29.9,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 7,
          "SampleId": 200698694,
          "Value": 30.4,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 6,
          "SampleId": 200698694,
          "Value": 30.5,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 5,
          "SampleId": 200698694,
          "Value": 29.8,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 4,
          "SampleId": 200698694,
          "Value": 30.8,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 2,
          "SampleId": 200698694,
          "Value": 30.7,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        },
        {
          "Seqnr": 1,
          "SampleId": 200698694,
          "Value": 29.5,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:32:57Z",
          "SampleTimestampUtc": "2023-10-06T07:32:57Z",
          "CreatedTimestamp": "2023-10-06T15:32:57Z",
          "CreatedTimestampUtc": "2023-10-06T07:32:57Z",
          "UpdatedTimestamp": "2023-10-06T15:32:57Z",
          "UpdatedTimestampUtc": "2023-10-06T07:32:57Z",
          "NumViolations": 0,
          "SampleMean": 30.2,
          "SampleMin": 29.5,
          "SampleMax": 30.8,
          "SampleMedian": 30.2,
          "SampleStdev": 0.465985898008574,
          "SampleQ1": 29.875,
          "SampleQ3": 30.55,
          "SampleSize": 8
        }
      ],
      "ParameterClass": "NON_PPAP",
      "Segment": "DS",
      "Module": "M1_ASSY",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 38.0,
          "spec_low": 19.0,
          "target": 28.5
        },
        "ControlLimits": {
          "ctrl_mean_low": 24.8896233993906,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 28.9572041233868,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 33.024784847383,
          "CtrlMeanHighEnabled": "Y",
          "ctrl_low": 24.0000978038884,
          "CtrlLowEnabled": "Y",
          "ctrl_target": 28.9570619469026,
          "ctrl_high": 33.9140260899169,
          "CtrlHighEnabled": "Y",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "Y",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "Y",
          "ctrl_stdev_target": 0.583862693698425,
          "ctrl_stdev_high": 2.15608484464821,
          "CtrlStdevHighEnabled": "Y",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 1.70352362448543,
          "ctrl_range_high": 6.21103188423075,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 8,
        "base_count": 8,
        "flagged_count": 0,
        "mean": 30.2,
        "median": 30.2,
        "stdev": 0.43588989435406733,
        "range": 1.3000000000000007,
        "min": 29.5,
        "max": 30.8,
        "q02": 29.542,
        "q05": 29.604999999999997,
        "q25": 29.874999999999996,
        "q75": 30.55,
        "q95": 30.765,
        "q98": 30.785999999999998,
        "NumViolations": 0,
        "Samples": "200698694"
      },
      "ParameterName": "Ball Height",
      "ParameterUnit": "um",
      "ChannelId": "25175",
      "SourceDataLevel": "C",
      "channel_name": "Ball_Height SOD323 Au 38um",
      "channel_description": "Created from LDS  exclude list by  action 'Create SPC Channel'",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-06T07:32:57Z",
      "created_timestamp": "2023-10-06T15:32:57Z",
      "updated_timestamp_utc": "2023-10-06T07:32:57Z",
      "updated_timestamp": "2023-10-06T15:32:57Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-10-06T07:32:57Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:WUX:BE:ACWUS:D_WireBond:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:VC327640C02",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "WUX:BE:VC327640C02:ACWUS:D_WireBond:Ball Height:25175:200698694:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-06T15:32:57Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:VC327640C02:SPACEAGGED2:WUX:BE:ACWUS:D_WireBond::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "WUX"
  }
}